package.json

```diff
@@ -145,6 +145,6 @@
     "fsevents": "^2.0.7"
   },
   "engines": {
-    "node": ">=13.2.0"
+    "node": ">=12.0.0"
```

Comment:
We're now accepting lower versions of node? 🤔

Reply:
not part of this PR, but looked up "why-did-you-update" dependency because I was curious what it did. Looks like its deprecated though: if there's a ticket for updating dependencies for portal-ui, could probably add this

Reply:
i can revert this. however, it's a bad practice to depend on odd versions of node.  those are basically development versions. the even version numbers are the stable and lts versions.  going down to 12 was easier than going up to 14. i'll create a separate ticket for updating the node version.  there's other things to be updated, such as docker.